flyteplugins/go/tasks/plugins/k8s/ray/ray.go

```diff
@@ -219,7 +219,40 @@ func buildHeadPodTemplate(container *v1.Container, podSpec *v1.PodSpec, objectMe
 
 	headPodSpec := podSpec.DeepCopy()
 
-	headPodSpec.Containers = []v1.Container{*primaryContainer}
+	// Ray logs integration
+	foundTmpRayVolMount := false
+	for _, vm := range primaryContainer.VolumeMounts {
+		if vm.MountPath == "/tmp/ray" {
```

Comment:
Is that path hardcoded in ray driver?

Reply:
Yea `/tmp/ray` is a fixed path. I'll make this a const.